.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,44 @@
+name: continuous-integration
+on: [pull_request]
+jobs:
+  jasmine-tests:
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+        check-latest: true
+    - run: npm install
+    - run: npm install grunt grunt-cli
+    - run: ./node_modules/grunt-cli/bin/grunt build
+    - run: ./node_modules/grunt-cli/bin/grunt transpile
+    - run: ./node_modules/grunt-cli/bin/grunt jasmine
+
+  cypress-tests:
+    runs-on: ubuntu-16.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+        check-latest: true
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+    - run: npm install
+    - run: npm run start:ci &
+    - uses: cypress-io/github-action@v2
+        with:
+          # just for full picture after installing Cypress
+          # print information about detected browsers, etc
+          # see https://on.cypress.io/command-line#cypress-info
+          build: npx cypress info
```

Comment:
This didn't work... I think we need to better configure cypress to get past the undefined takeScreenshot error from the previous  commit...